example/ck_tile/12_smoothquant/script/perf_test.sh

```diff
@@ -0,0 +1,38 @@
+
+# run from top of ck folder
+EXE=build/bin/tile_smoothquant
```

Comment:
Not everyone name the build directory exactly as `build`, I suggest to use `find` command to find 
available executable to run (under current folder)
```bash
EXE="$(find . -name tile_smoothquant -type f | head -n 1)"
```

Reply:
e9dc1a4af35c0da075649b85a11cebec0b18b6b8